src/bundle/Command/DumpSqlCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\DoctrineSchema\Command;
+
+use Doctrine\DBAL\Connection;
+use Ibexa\DoctrineSchema\Builder\SchemaBuilder;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+final class DumpSqlCommand extends Command
+{
+    protected static $defaultName = 'ibexa:schema:dump-sql';
```

Comment:
`schema` is too generic IMO. Can refer also to GraphQL and maybe a few more.

- `ibexa:doctrine:schema:dump-sql`
- `ibexa:database:schema:dump-sql`
- `ibexa:db:schema:dump-sql`
- ... ?